### ISSUE TITLE
Property IsProject does not exist. Class IPPCustomer

### ISSUE THREAD

**Issue by Abhilash-k-p** [intuit/QuickBooks-V3-PHP-SDK#274]
When i am trying to add cutomer to Quickbooks using following code.

`$dataService->Add($customerObj);`

I am getting the following exception in **sendRequestParseResponseBodyAndHandleHttpError** internal function of sdk.

* **Property IsProject does not exist. Class IPPCustomer**

**Comment by bsivalingam** [comment]
@Abhilash-k-p Can you please update your SDK to use the latest vesion?

**Comment by KelseySheely** [comment]
I am receiving this error on the 5.3.4 branch.

**Comment by bsivalingam** [comment]
That is super odd, can you please share the request? I am on the latest branch and I don't see the error. Can you please check your code in vendor directory to see if you have this line of code: https://github.com/intuit/QuickBooks-V3-PHP-SDK/blob/master/src/Data/IPPCustomer.php#L525 ?

**Comment by KelseySheely** [comment]
Yes, I had found that the library does in fact have the IsProject property on the IPPCustomer class. I'm thinking it must be conflicting with a completely different ancient forked version of the SDK that is on the server which also has an IPPCustomer class. Although that's strange because it is namespaced totally differently. I ran out of time this afternoon, but I'll debug some more tomorrow and let you know if that is the problem.

**Comment by KelseySheely** [comment]
Removing the other SDK did fix the problem. 

This is a problem because the XML Serializer prioritizes the non-namespaced IPP class rather than the namespaced class. I'm not sure if there is a reason for that, but it seems to me it shouldn't ever be looking for a class outside of the namespace:

The issue is in src/Core/Http/Serialization/XmlObjectSerializer.php in the PhpObjFromXml method starting on line 86:

        if (class_exists($className, CoreConstants::USE_AUTOLOADER)) {
            $phpObj = new $className;
        } elseif (class_exists(CoreConstants::NAMEPSACE_DATA_PREFIX . $className, CoreConstants::USE_AUTOLOADER)) {
            $className = CoreConstants::NAMEPSACE_DATA_PREFIX . $className;
            $phpObj = new $className;
        } else {
            throw new \Exception("Can't find corresponding CLASS for className" . $className . "during unmarshall XML into POPO Object");
        }

I propose that the first 'if' block should be completely removed. At the very least, it should come after the first elseif.

**Comment by bsivalingam** [comment]
@KelseySheely calling the issue closed, you can send a PR with the changes, we can evaluate and merge. Thanks!
